app/styles/play/play-level-view.sass

```diff
@@ -6,6 +6,7 @@ body.is-playing
   background-color: black
 
 $level-resize-transition-time: 0.5s
+// TODO: use this for real-time playback but not cinematic playback
```

Comment:
Is this TODO for `$level-resize-transition-time`?  If so, should probably but comment above or to right.  Following like this seems non-standard.